Vagrantfile

```
@@ -78,13 +87,17 @@ cd /vagrant # This is where the host folder/repo is mounted
 gem install bundler foreman
 bundle install
 
-# Install node modules
+# Disable husky inside vagrant:
+echo "HUSKY=0" >> ~/.bash_profile
+
+# Install node modules, uses:
+# --ignore-scripts to prevent scripts automatically executing inside vagrant
+# --frozen-lockfile to prevent lockfile updates coming from vagrant
 sudo corepack enable
-yarn set version classic
-yarn install
+yarn install --ignore-scripts --frozen-lockfile
```

The ignore scripts here will conflict with #24702 which uses a `postinstall` script to install the streaming server, so we'll need this change here: 
```suggestion
yarn install --ignore-scripts --frozen-lockfile
cd streaming && yarn install --ignore-scripts --frozen-lockfile
```